nemo_aligner/models/nlp/gpt/megatron_gpt_ppo_actor.py

```diff
@@ -263,6 +310,10 @@ def prepare_for_inference(self):
         set_eval(self)
         self.offload_adam_states()
 
+        if self.use_trtllm_generation:
+            self.trtllm_generate.refit(self.model)
```

Comment:
ah that's a good point. but because this happens relatively rarely(since it's during validation) and refit is decently fast(3s) can we leave this for another PR?

Reply:
we definitely need to do this though if we have onload/offload engines cc @terrykong for FYI

Reply:
Ok, I thought it was slower than that, 3s doesn't sound too bad. Can you add a comment though, something like:
```python
# TODO this might be optimized to avoid calling `refit()` twice in a row after a validation step
```

Reply:
sounds good! 4ffc70690c286ed8dc33e80d5af6b90ef5d70a4a